wsd/COOLWSD.cpp

```diff
@@ -3824,6 +3824,7 @@ int COOLWSD::innerMain()
 #if defined(M_TRIM_THRESHOLD)
     LOG_DBG("trimming memory post startup");
     malloc_trim(0);
+    time_t prevTrimTrigger = 0;
```

Comment:
I think we should try to use std::chrono everywhere for time - and ideally the steady_clock which copes with discontiguous time too - but I see we have lots of time_t in the Admin code already.